demo/api/schema.gql

```diff
@@ -748,6 +748,7 @@ type Query {
   pageTreeNodeList(scope: PageTreeNodeScopeInput!, category: String): [PageTreeNode!]!
   paginatedPageTreeNodes(scope: PageTreeNodeScopeInput!, category: String, sort: [PageTreeNodeSort!], documentType: String, offset: Int! = 0, limit: Int! = 25): PaginatedPageTreeNodes!
   pageTreeNodeSlugAvailable(scope: PageTreeNodeScopeInput!, parentId: ID, slug: String!): SlugAvailability!
+  sitePreviewJwt(scope: JSONObject!, path: String!, includeInvisible: Boolean!): String!
```

Comment:
imho this should be a mutation, not a query:

- it creates something new (although not stored, so it strictly speaking does not mutate a database or something)
- it should never be cached by anything, that might even result into security issues

Reply:
We could call it createSitePreviewJwt or generateSitePreviewJwt